static_files/css/style.css

```diff
@@ -7,7 +7,34 @@ html {
 }
 
 @import url('https://fonts.googleapis.com/css2?family=Dongle:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Chenla&family=Dongle:wght@300&display=swap');
 
 body{
     font-family: "Dongle", sans-serif;
+    min-height: 100vh;
 }
+
+header{
+    min-height: 50px;
+}
+
+body{
+    display: flex;
+    flex-direction: column;
+}
+
+main{
+    flex: 1;
+}
+
+.Footer {
+    min-height: 50px;
+    font-family: "Nunito", sans-serif;;
+    text-align: center;
+    color: #CCCCCC;
+    font-size: small;
+    padding: 2% 0;
```

Comment:
I find it interesting to set the footer padding to a fixed value, for example, in "px", as this avoids layout differences on devices with different resolutions. Note the current padding behavior in the attachments below. It is possible to observe that when the device is in low resolution there is almost no padding footer, contrary to a device with high resolution. This behavior seems to be very strange. If this behavior is not predicted by the PD, I suppose it is not interesting to do this.

![image](https://user-images.githubusercontent.com/105747728/232601534-628daecc-c07e-48ba-b9af-ef170127aac9.png)

![image](https://user-images.githubusercontent.com/105747728/232601595-684ef2c0-2abb-4f8b-bd4f-0a396006eae2.png)